.rubocop_local.yml

```diff
@@ -46,6 +46,6 @@ Rails/Exit:
 Rails/Output:
   Exclude:
   - lib/workers/bin/*
-Style/ExtraSpacing:
+Layout/ExtraSpacing:
```

Comment:
I don't see a "Layout" in rubocop v0.47.1

Reply:
I reverted the commit that did the rubocop changes.
Sorry, I used rubocop v0.51.0